forgescan_realsense/srv/CameraPose.srv

```diff
@@ -1,3 +1,4 @@
 geometry_msgs/Pose pose
 ---
-EigenVector[] eigenmatrix
\ No newline at end of file
+EigenVector[] eigenmatrix
+int64 length
```

Comment:
Why did we add length? I understand the EigenVector[] is the eigenmatrix for the camera intrinsics, but what would I use the length variable for?